### PR TITLE
Implement pixel operation 'to monochrome'

### PR DESCRIPTION
**Pull request by ivancho-ifa**
I branched it from dev/issue-20. Gonna submit after dev/issue-20 is submitted. A little bit ugly pull-request. I didn't manage to fix the diffs. Please review dev/issue-20 first.

### CODE REVIEW

File: build/raster-cli/raster-cli-tests/raster-cli-tests.vcxproj.filters [hunk 1]
```diff
@@ -36,12 +45,21 @@
...
+      <Filter>Source Files\utils\ToMonochrome</Filter>
+    </ClCompile>
+    <ClCompile Include="..\..\..\test\impl\utils\RGBPixel\RGBPixelConstants.cpp">
       <Filter>Source Files\utils\RGBPixel</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\..\test\impl\utils\RGBPixel\Helpers.cpp">
+    <ClCompile Include="..\..\..\test\impl\utils\RGBPixel\RGBPixelHelpers.cpp">
       <Filter>Source Files\utils\RGBPixel</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\..\test\impl\utils\ToMonochrome\ToMonochromeHelpers.cpp">
```
Comment (nikolovay) [inline build/raster-cli/raster-cli-tests/raster-cli-tests.vcxproj.filters]: I don't like this folder nested and all folders to contains files with folder name prefix. I think we have to discuss this offline.

File: test/impl/utils/RGBPixel/RGBPixelHelpers.cpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#include <utils/RGBPixel/Helpers.hpp>
+
+#include <pixels/RGBPixel.hpp>
+
+using boost::test_tools::predicate_result;
+
+using std::ostream;
+
+bool operator==(const RGBPixel& pixel_a, const RGBPixel& pixel_b) noexcept
```
Comment (nikolovay) [inline test/impl/utils/RGBPixel/RGBPixelHelpers.cpp]: This can be into the RGBPixel file

**Review by nikolovay — APPROVED** [review]

File: build/raster-cli/raster-cli-tests/raster-cli-tests.vcxproj.filters (hunk 1 above)
Comment (ivancho-ifa) [reply build/raster-cli/raster-cli-tests/raster-cli-tests.vcxproj.filters]: Doing impl/utils/Operations/Helpers.cpp

File: test/impl/utils/RGBPixel/RGBPixelHelpers.cpp (hunk 2 above)
Comment (ivancho-ifa) [reply test/impl/utils/RGBPixel/RGBPixelHelpers.cpp]: Doing it.

**Pull request merged**